crypto/mbedtls/syscfg.yml

```diff
@@ -125,6 +125,9 @@ syscfg.defs:
   MBEDTLS_ENTROPY_C:
     value: 1
 
+  MBEDTLS_ENTROPY_HARDWARE_ALT:
+    value: 0
```

Comment:
it would not hurt to provide description field, but since this file lack any descriptions I don't mind after consideration